$NetBSD: patch-ak,v 1.2 2012/12/07 23:52:09 schnoebe Exp $

pkgsrc places IDN's stringprep in a different directory than other
package managers.

--- iris/src/xmpp/jid/jid.cpp.orig	2012-10-04 00:53:09.000000000 +0000
+++ iris/src/xmpp/jid/jid.cpp
@@ -24,7 +24,7 @@
 #include <QCoreApplication>
 #include <QByteArray>
 #include <QHash>
-#include <libidn/stringprep.h>
+#include <stringprep.h>
 
 #ifndef NO_IRISNET
 #include "irisnetglobal_p.h"
